src/goTest.ts

```diff
@@ -112,6 +140,22 @@ export function testCurrentFile(timeout: string) {
 }
 
 /**
+ * Kill all running go task.
+ *
+ */
+export function killRunning() {
```

Comment:
Kind of scary to do a blanket kill with a pattern match. Would be far safer to track PIDs if anything.

@lukehoban, process management seems like something VScode or Electron should provide some facilities for. Any clue?